### DISCUSSION TITLE
Gestion des serveurs cloud

### DISCUSSION THREAD

**Discussion by webchl** [dbarzin/mercator#405] [Q&A]
Bonjour a tous,

comment gérer vous les serveurs dans le cloud? 
je m'explique: Comment enregistrez vous, comment rattacher vous une application qui est hébergée dans le cloud sur des serveurs (donc logiques et physiques) inconnu de l'infra?
merci

**Comment by dbarzin** [comment]
Tu ne modélises pas ces assets car ce n'est pas ton infrastructure. C'est la même chose que pour l'application bancaire utilisée par ton organisation pour faire des paiements en ligne. Tu ne modélises pas l'infrastructure de la banque.

**Reply by webchl** [reply]
ok mais nous avons tout de meme des relations avec les prestataires et des flux entre ces divers applications.
comment les faire ressortir?
merci

**Comment by dbarzin** [comment]
Un flux peut avoir une application comme destination.

**Reply by webchl** [reply]
et donc l'application de destination du flux est hébergé sur un serveur logique non référencé?
quel est la bonne pratique?

**Comment by dbarzin** [comment]
Une application ne doit pas nécessairement avoir un serveur logique.

**Reply by webchl** [reply]
OK merci +++
